### PR TITLE
added information to abs function

### PR DESCRIPTION
**Pull request by shilpi03**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/python/abs-function/index.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
 ---
 title: Python Abs Function
 ---
-`abs()` is a built-in function in Python 3, to compute the absolute value of any number. The absolute value of a number "means only how far a number is from 0" <sup>1</sup> It takes one argument `x`. The argument can even be a <a href='https://docs.python.org/3.0/library/cmath.html' target='_blank' rel='nofollow'>complex number</a>, and in that case its <a href='http://www.mathcentre.ac.uk/resources/sigma%20complex%20number%20leaflets/sigma-complex9-2009-1.pdf' target='_blank' rel='nofollow'>modulus</a> is returned.
+`abs()` is a built-in function in Python 3, to compute the absolute value of any number. The absolute value of a number "means only how far a number is from 0" <sup>1</sup> It takes one argument `x`. The argument can be an integer, a floating point value or a  <a href='https://docs.python.org/3.0/library/cmath.html' target='_blank' rel='nofollow'>complex number</a>, in case of complex number its <a href='http://www.mathcentre.ac.uk/resources/sigma%20complex%20number%20leaflets/sigma-complex9-2009-1.pdf' target='_blank' rel='nofollow'>modulus</a> is returned.
```
Comment (thecodingaviator) [inline guide/english/python/abs-function/index.md]: Change the link here to markdown format

File: guide/english/python/abs-function/index.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 
 ## Argument
 
-It takes one argument `x` - an integer, or decimal, or a complex number.
+It takes one argument `x` - an integer, or float, or a complex number.
+
+## Syntax
+
+abs(x)
```
Comment (thecodingaviator) [inline guide/english/python/abs-function/index.md]: Change this to code format

File: guide/english/python/abs-function/index.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
 
 ## Return Value
 
-The return value would be a positive number. Even if complex number is passed, it would return its magnitude, computed as per complex number algebra.
+The return value is an integer or a float. Even if complex number is passed, it would return its magnitude, computed as per complex number algebra.
```
Comment (thecodingaviator) [inline guide/english/python/abs-function/index.md]: Make this what it was originally, it is more clear

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Please clear merge conflicts and check comments below

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Check comments above, same thing

**Comment by RandellDawson** [comment]
@thecodingaviator This one has gone stale.

**Comment by thecodingaviator** [comment]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**